.changeset/neat-schools-take.md

```diff
@@ -0,0 +1,12 @@
+---
+"@spectrum-css/clearbutton": major
+---
+
+#### S2 migration for clear button
+
+This migrates the clear button component to the latest Spectrum 2 designs. Custom properties have been remapped and added per the design specification.
+
+- The quiet variant has been remove as all clear buttons have a transparent background.
+- The over-background and static color variants have been removed.
```

Comment:
```suggestion
- The `over-background` and `static color` variants have been removed.
```

What does this translate to in the code? I like this general note but might be good to translate this into, _X_ class was removed for example.